example/hap.yaml

```diff
@@ -0,0 +1,9 @@
+deploy_path: "/tmp/hap-example"
+repo: "https://github.com/stackbuilders/hapistrano.git"
+revision: "origin/add_working_directory"
```

Comment:
is this the desired value?

Reply:
No, it's just a placeholder for testing purposes. The final value will be the `master` branch.

Reply:
Reminder that this value has not been updated